src/components/ProfileModal.tsx

```diff
@@ -1,9 +1,14 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import Input from '@site/src/ui/design-system/src/lib/Components/Input';
 import Field from '@site/src/ui/design-system/src/lib/Components/Field';
 import Modal from '@site/src/ui/design-system/src/lib/Components/Modal';
 import RadioGroup from '@site/src/ui/design-system/src/lib/Components/RadioGroup';
 import { Button } from '@site/src/ui/design-system/src/lib/Components/Button';
+import { ProfileSettings, SocialType } from '../types/gold-star';
+import { useProfile } from '../hooks/use-profile';
+import { useCurrentUser } from '../hooks/use-current-user';
+import { createProfile, setProfile } from '../utils/gold-star';
+import _ from 'lodash';
```

Comment:
Do you want to just import the things you need for tree shaking? e.g. `import { isEqual } `